### ISSUE TITLE
silvermoon emissary walk

### ISSUE THREAD

**Issue by daunte5386** [dalaranwow/dalaran-wow#5655]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Where: Silvermoon City
When: All the time
What: The horde emissary that marches through Silvermoon do not move and have stopped at the entrance to the crafting zone area
Why: Full emersion
Who: Dela Runetotem, Kristine Denny, Tatai, Cheneta, Ambassador Kelemar

**Expected behaviour**: Tell us what should happen instead
Expected Behavior: Ambassador Kelemar should be escorting the Horde Emissaries around Silvermoon as of right now they are just standing in front of the Farstrider Square entrance. He should have certain dialogue when he escorts them to various places.

**Steps to reproduce the problem**:

1.  Enter Silvermoon City
2. Go to The Royal Exchange and walk till you get to the entrance area of Farstriders Square
3. The emissaries are standing at the entrance area

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

https://www.wowhead.com/npc=18191/ambassador-kelemar#comments
